packages/vue/src/tracing.ts

```diff
@@ -46,6 +46,20 @@ function finishRootSpan(vm: VueSentry, timestamp: number, timeout: number): void
   }, timeout);
 }
 
+// Find if the current component exists in the provided `TracingOptions.trackComponents` array option.
+function findTrackComponent(trackComponents: string[], formattedName: string): boolean {
+  const isMatched = trackComponents.some(compo => {
+    let currentCompo = compo;
+
+    if (!(currentCompo.startsWith('<') && currentCompo.endsWith('>'))) {
+      currentCompo = `<${currentCompo}>`;
+    }
+    return formattedName === currentCompo;
```

Comment:
Ah in this case we don't even need the block anymore but could inline it to the `some` call. So feel free to do that as well :)

Reply:
Noted

Reply:
hey, I've taken your approach and found that `formattedName` must also be replaced: 

```javascript
function findTrackComponent(trackComponents: string[], formattedName: string): boolean {

  function extractComponentName(name: string): string {
    return name.replace(/^<(.*)>(?:.*)?$/, "$1");
  }
  const isMatched = trackComponents.some(compo => {
    return    extractComponentName(formattedName) === extractComponentName(compo)
  }
  );
  return isMatched;
}
```
The regex is needed for both the user provided component and the formatted component. 
- User provided: either `<App>` or `App`
- Formatted: either `<App>` or `<App> at App.vue`.

What do you think?

Reply:
Good observation, you're right! Also, I didn't think about the `<App> at App.vue` case. 

There's one unfortunate side effect of this regex though: It can lead to polynomial build up when evaluating an input. Given that we run `extractComponentName` on user input I'd rather not risk the (still unlikely but possible) chance of some kind of ReDoS attack (I used this [ReDoS checker](https://devina.io/redos-checker) to test it) 

I played around with the regex to make it safer and keep it linear and arrived at this one: `/^<([^\s]*)>( at [^\s]*)?$`
I think this _should_ still cover all component names but would appreciate it if you could double check it as well :) 

Also, since this logic is getting a little bit more involved now, could you add a couple of unit test for `findTrackComponent`? It's fine to export the function just for testing purposes btw. Thanks!

(I don't want to overburden you with tests btw, I just want to keep stuff like this covered and understandable for everyone. So if you don't have the time, just let me know. We appreciate your hard work either way!)

Reply:
Noted, unit and e2e tests added.